### ISSUE TITLE
Materialboxed slave to column...

### ISSUE THREAD

**Issue by TheAdamGalloway** [Dogfalo/materialize#1116]
I just upgraded to 0.96.1 and tried Materialboxed again... While it was originally blocked by the card that it was declared under, it is now blocked by the column instead. Removing the column fixes it, but ruins my formatting.

**Comment by Dogfalo** [comment]
Take a look at our showcase to see an example of how we do it

**Comment by TheAdamGalloway** [comment]
Yeah I had a look at your showcase, but it appears that it's a different issue. I have a materialbox inside a card inside a column inside a row. I looked extensively through the CSS and found that the issue was that the overflow of the card is set to hidden... 

Should I just remove this for that specific element or what?

**Comment by Dogfalo** [comment]
If your card doesn't use card-reveal then you can probably remove it

**Comment by TheAdamGalloway** [comment]
I removed it and it worked fine. I do have another card that uses card-reveal which is having a similar issue, in which the card content contains an expandable image. When clicked, the entire screen is filled by materialbox, but no image is present.
